src/GameServer/Systems/Player/MapMarksManager.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Vim.Math3d;
+using Weedwacker.GameServer.Packet.Send;
+using Weedwacker.Shared.Network.Proto;
+using static Weedwacker.GameServer.Systems.Script.Scene.DummyPoints;
+using static Weedwacker.Shared.Network.Proto.MarkMapReq.Types;
+
+namespace Weedwacker.GameServer.Systems.Player
+{
+    internal class MapMarksManager
+    {
+        private Player player;
```

Comment:
c# fields start with an uppercase letter. Visual Studio usually informs you so if you hover over the name. Also, `Owner` remains consistent with the other managers.
```suggestion
        private readonly Player Owner;
```